conf/cassandra.yaml

```diff
@@ -1111,6 +1111,13 @@ compaction_large_partition_warning_threshold_mb: 100
 # as corrupted. This should be positive and less than 2048.
 # max_value_size_in_mb: 256
 
+# If replication factor is not mentioned as part of keyspace creation, default_keyspace_rf would apply.
+# This would also apply for any system keyspaces that need replication factor.
+#default_keyspace_rf: 1
+
+# Indicates the minimum replication factor that a keyspace should have.
```

Comment:
I'd make it a little clearer that this will prevent keyspaces with simple strategy from being created.
" The minimum allowable replication factor. Creating a keyspace with a replication factor less than this value will be rejected"